radio/src/myeeprom.h

```diff
@@ -422,7 +422,8 @@ PACK(typedef struct {
   int8_t  rfProtocol:4;
```

Comment:
I can't remember why it is an int rather than an uint, and I am not sure we use more than 7 rfProtocols today. If I am not mistaken, this could be changed to an uint8_t